common/config/rush/pnpm-config.json

```diff
@@ -13,6 +13,7 @@
     "rollup-plugin-copy>globby": "^11.0.0", // https://github.com/vladshcherbin/rollup-plugin-copy/issues/77
     "elliptic": "^6.6.1", // https://github.com/advisories/GHSA-vjh7-7g9h-fjfh crypto-browserify>browserify-sign>elliptic
     "form-data": "^4.0.4", // https://github.com/advisories/GHSA-fjxv-7rqg-78g4 azurite>@azure/ms-rest-js>form-data
+    "glob": "^10.5.0", // https://github.com/advisories/GHSA-5j98-mcp5-4vw2 cpx2>glob, vitest/coverage-v8>glob
```

Comment:
How many packages require override ? If it's only a few, I would suggest adding "scoped" overrides, since our dependencies seem to be using a variety of `glob` versions, include a newer 11